### ISSUE TITLE
Crowdin Markdown table+pipe issue

### ISSUE THREAD

**Issue by slorber** [facebook/docusaurus#5602]
## 🐛 Bug Report

We use pipe in markdown tables to describe TS union types

```
```

![image](https://user-images.githubusercontent.com/749374/134309503-1fca8f3c-0524-4547-866d-d484326b232b.png)

**Comment by slorber** [comment]
The Crowdin team is on it, they could reproduce the bug

**Comment by forresst** [comment]
@slorber Do you have news from Crowdin?

**Comment by slorber** [comment]
hey, yes forgot but got an answer:

![image](https://user-images.githubusercontent.com/749374/148373885-3d797781-f4c3-4cdf-bf6f-67333907336e.png)

**Comment by Josh-Cena** [comment]
Indeed, we don't have `<small>` now 😄 But I don't know if they actually mean `<small>` or `<code>`...

**Comment by forresst** [comment]
It is true that Crowdin's answer is a bit ambiguous.

How can we proceed to test Crowdin's workaround?

**Comment by slorber** [comment]
maybe we revert some changes of the linked PR and see if it works better after merge?

**Comment by forresst** [comment]
For your information, I found a workaround in Crowdin while waiting for Crowdin's correction

You just have to copy the original English version with the `<code>` and `</code>` tags

![image](https://user-images.githubusercontent.com/163352/155320227-e623fde5-e6cb-4aa3-b99a-119e269ce0cf.png)

The rendering in french on docusaurus.io is [now correct](https://docusaurus.io/fr/docs/next/api/plugins/@docusaurus/plugin-content-docs#configuration) : 

![image](https://user-images.githubusercontent.com/163352/155321161-a1cc592a-57f7-41b1-9186-3f146121da47.png)

**Comment by Josh-Cena** [comment]
@forresst Could you elaborate on the "copy the original English version"? Do you mean checking out the original Markdown source? Because the "Copy Source" button still copies gibberish.

Re-typing the `\|` certainly works, because it's Markdown after all.
